JS/Airport.js

```diff
@@ -110,8 +91,6 @@ class Airport {
     }
 
 
-
-
     static print(planes) {
```

Comment:
This method makes the Airport class "magic".
Please remove it from here and create a separate function for this purpose.

Reply:
Done